candle-examples/examples/falcon/main.rs

```diff
@@ -36,9 +36,8 @@ struct Args {
     revision: String,
 }
 
-#[tokio::main]
-async fn main() -> Result<()> {
-    use candle_hub::{api::Api, Repo, RepoType};
+fn main() -> Result<()> {
```

Comment:
You may want to rebase as I pushed some fixes for this too yesterday evening: https://github.com/LaurentMazare/candle/pull/97/files